### PR TITLE
HPCC-16701 Determine optimal set of default metrics on Ganglia plugin

### PR DESCRIPTION
**Pull request by kunalaswani**
@rpastrana please code review

### CODE REVIEW

**Comment by rpastrana** [comment]
@kunalaswani looks reasonable.

**Comment by kunalaswani** [comment]
@rpastrana thanks.

@richardkchapman this is targeted at master, is that correct or should we create a 6.4 branch?

**Comment by richardkchapman** [comment]
master is ok - it will ned up in 6.4.0 when we do split that off.

File: esp/src/ganglia/ganglia.json [hunk 1]
```diff
@@ -4,55 +4,39 @@
...
-		["totScanLength", "meanScanLength", "totScans"],
-		["restarts"],
-		["activitiesStarted", "globalLocks", "retriesNeeded"],
-		["leafCacheAdds", "leafCacheHits", "cacheHits", "preloadCacheAdds"],
-		["nodesLoaded"] 
-	],
-	"time":["Hour", "Day", "Week", "Month"]
+        "name": "Performance and Health",
+        "metrics" :[
+                ["totScanLength", "meanScanLength", "totScans"],
```
Comment (richardkchapman) [inline esp/src/ganglia/ganglia.json]: I don't think these stats are useful. In fact I may create a Jira to remove these stats altogether

**Comment by richardkchapman** [comment]
@kunalaswani Back to you

File: esp/src/ganglia/ganglia.json (hunk 1 above)
Comment (kunalaswani) [reply esp/src/ganglia/ganglia.json]: To be clear, do you want me to remove "totScanLength", "meanScanLength", "totScans"?

Comment (richardkchapman) [reply esp/src/ganglia/ganglia.json]: Yes

**Comment by kunalaswani** [comment]
@richardkchapman please review latest commit

**Pull request merged**